### PR TITLE
Add Apptim

### PR DESCRIPTION
**Pull request by morellexf13**
## Requirements

 * [✓] This is Software as a Service not self hosted
 * [✓ ] It has a free tier not just a free trial
 * [✓] The submission mentions what is free
 * [✓] The submission is not already present in the list
 * [✓] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -557,6 +557,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
   * [Applitools.com](https://applitools.com/) — Smart visual validation for web, native mobile and desktop apps. Integrates with almost all automation solutions (like Selenium and Karma) and remote runners (Sauce Labs, Browser Stack). free for open source. A free tier for a single user with limited checkpoints per week.
   * [Appetize](https://appetize.io) — Test your Android & iOS apps on this Cloud Based Android Phone/Tablets emulators and iPhone/iPad simulators directly in your browser. Free tier includes 1 concurrent session with 100 minutes usage per month. No limit on app size.
+  * [Apptim](https://apptim.com) — A mobile performance testing tool that empowers mobile developers and testers to effortlessly validate the performance and user experience (UX) of their Android and iOS apps. It efficiently captures and analyzes device resource usage, including CPU, Memory, FPS, Network, Crashes, Render Times, and more (FREE). The tool seamlessly facilitates running automated tests on real devices in the cloud, while offering seamless integration with CI/CD workflows (PAID).
```
Comment (ripienaar) [inline README.md:560]: The free tier is only with the desktop app and so runs tests locally and not in the SaaS right?

Then I don’t think this is suited to the list. 

The entry is also just way too wordy and full of marketing speak

Comment (ripienaar) [reply README.md:560]: Question isn’t about user counts. Question is where does tests execute? On the SaaS or on your desktop?

Comment (fbaptista) [reply README.md:560]: Hi 👋🏼, Apptim CTO here. 
Tests are manual, they run on the local phone and results stored on desktop. Then test results are published, shared, and KPI / trends are hosted on the cloud (SaS). This is the free offering. 
If you want to launch CICD automated tests in your own infrastructure or AWS then you need to use Apptim CLI approach (Paid).

Comment (ripienaar) [reply README.md:560]: Ok that sounds reasonable thanks for the explanation. 

Your proposed description seems fine please update the PR.

Comment (fbaptista) [reply README.md:560]: I see this mechanism the same as the [BugBug](https://bugbug.io/) - Lightweight test automation tool for web applications. Easy to learn and doesn't require coding. You can **run unlimited tests on your own computer for free.** For an additional monthly fee, you also get cloud monitoring and CI/CD integration.

Comment (fbaptista) [reply README.md:560]: > Ok that sounds reasonable thanks for the explanation.
> 
> Your proposed description seems fine please update the PR.

Excellent, thanks! @morellexf13

Comment (morellexf13) [reply README.md]: Done @fbaptista @ripienaar

**Comment by ripienaar** [comment]
Thank you
